src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    static int numberOfTasks = 0;
```

Comment:
Maybe numberOfTasks can be protected variable too, so that it can only be acceessed by get method:)